episodes/operations.md

```diff
@@ -6,96 +6,61 @@ exercises: 2
-(with extension `.Rmd`). Please refer to the [Introduction to The Carpentries
-Workbench][carpentries-workbench] for full documentation.
-
-What you need to know is that there are three sections required for a valid
-Carpentries lesson template:
-
- 1. `questions` are displayed at the beginning of the episode to prime the
-    learner for the content.
- 2. `objectives` are the learning objectives for an episode displayed with
-    the questions.
- 3. `keypoints` are displayed at the end of the episode to reinforce the
-    objectives.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+- describe the lesson life cycle 
+  and the pathways a lesson can follow in The Carpentries community.
+- prepare to teach Collaborative Lesson Development Training.
 
-Inline instructor notes can help inform instructors of timing challenges
-associated with the lessons. They appear in the "Instructor View"
-
-::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
-
-::::::::::::::::::::::::::::::::::::: challenge 
-
-## Challenge 1: Can you do it?
-
-What is the output of this command?
+::::::::::::::::::::::::::::::::::::::::::::::::
 
-```r
-paste("This", "new", "lesson", "looks", "good")
-```
 
-:::::::::::::::::::::::: solution 
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-## Output
- 
-```output
-[1] "This new lesson looks good"
-```
+## Reading List
 
-:::::::::::::::::::::::::::::::::
+- [CLDT episode: The Carpentries Workbench](https://carpentries.github.io/lesson-development-training/07-infrastructure.html)
+- FIXME
 
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Challenge 2: how do you nest solutions within challenge blocks?
 
-:::::::::::::::::::::::: solution 
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
 
-You can add a line with at least three colons and a `solution` tag.
+## Discussion Questions
 
-:::::::::::::::::::::::::::::::::
-::::::::::::::::::::::::::::::::::::::::::::::::
+- FIXME
 
-## Figures
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-You can use pandoc markdown for static figures with the following syntax:
 
-`![optional caption that appears below the figure](figure url){alt='alt text for
-accessibility purposes'}`
+:::::::::::::::::::::::::::::::::::::::::::::::::: challenge
 
-![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
+## Trainer Checkout
 
-## Math
+Do the following to complete your Trainer certification:
 
-One of our episodes contains $\LaTeX$ equations when describing how to create
-dynamic reports with {knitr}, so we now use mathjax to describe this:
+1. Submit a pull request to fix [an issue on the Collaborative Lesson Development Training curriculum][cldt-issues].
+2. Teach Collaborative Lesson Development Training alongside an experienced Trainer.
```

Comment:
Do you think they would need to teach all parts? Or could they teach in sections 1-4 or 5-6 to complete trainer certification?